packages/EasyStandard/docs/rectors/rectors.md

```diff
@@ -0,0 +1,180 @@
+---eonx_docs---
+title: 'EasyStandard: Rectors'
+weight: 2001
+---eonx_docs---
+##### [\EonX\EasyStandard\Rector\AddCoversAnnotationRector][1]
+Adds `@covers` annotation for test classes
+```php
+// before
+class SomeServiceTest extends \PHPUnit\Framework\TestCase
+{
+}
+```
+```php
+// after
+/**
+ * @covers \SomeService
+ */
+class SomeServiceTest extends \PHPUnit\Framework\TestCase
+{
+}
+```
+**Parameters**
+
+- `replaceArray` - An array of strings that will be cut from the FQCN when searching for the class covered by this test. Default value: `[]`.
+
+##### [\EonX\EasyStandard\Rector\AddSeeAnnotationRector][2]
+Adds `@see` annotation for data providers
+```php
+// before
+/**
+ * Provides some data.
+ * 
+ * @return mixed[]
+ */
+public function provideSomeData(): array
+{
+}
+```
+```php
+// after
+/**
+ * Provides some data.
+ * 
+ * @return mixed[]
+ * 
+ * @see testMethod
+ */
+public function provideSomeData(): array
+{
+}
+```
+
+##### [\EonX\EasyStandard\Rector\AnnotationsCommentsRector][3]
+Comments should have punctuation marks at the end of the sentence.
+```php
+// before
+/**
+ * Some class
+ */
+class SomeClass
+{
+}
+```
+```php
+// after
+/**
+ * Some class.
+*/
+class SomeClass
+{
+}
+```
+##### [\EonX\EasyStandard\Rector\ExplicitBoolCompareRector][4]
+Makes bool conditions more pretty
```

Comment:
"more pretty" -> "prettier" and add a full stop

Reply:
Done.